environment.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
       - pyarrow
       - pandas
       - --find-links https://release.colossalai.org colossalai
-      - --find-links https://download.pytorch.org/whl/cu113/torch_stable.html torch==1.11.1+cu113
+      - --find-links https://download.pytorch.org/whl/cu113/torch_stable.html torch==1.12.0+cu113
```

Comment:
torchaudio and torchvision need update too?
Not sure about using conda to install pytorch is a better idea?

Reply:
from torch 1.12, need `torch.backends.cuda.matmul.allow_tf32 = True` to use tf32 on A100. Maybe add this flag in fastfold and add note in README.md.